PUSH_TO_FILE.md

```diff
@@ -82,48 +291,60 @@ for a description of each environment variable setting:
 | `conjur.org/retry-interval-sec`     | `RETRY_INTERVAL_SEC`  | Defaults to 1 (sec)              |
 | `conjur.org/debug-logging`          | `DEBUG`               | Defaults to `false`              |
 | `conjur.org/conjur-secrets.{secret-group}`      | Push to File config is not available with environmental variables | List of secrets to be retrieved from Conjur. Each entry can be either:<ul><li>A Conjur variable path</li><li> A key/value pairs of the form `<alias>:<Conjur variable path>` where the `alias` represents the name of the secret to be written to the secrets file |
-| `conjur.org/secret-file-path.{secret-group}`    | Push to File config is not available with environmental variables | Path for secrets file to be written. <br> For the initial release of push-to-file the secret file path for the shared secrets must be '/conjur/secrets' . The file path must also include a file name (i.e. must not be a directory). Values ending with `/` are rejected and cause the Secrets Provider to abort.
+| `conjur.org/secret-file-path.{secret-group}`    | Push to File config is not available with environmental variables | Path for secrets file to be written. <br> The file path must include a file name (i.e. must not be a directory). Values ending with `/` are rejected and cause the Secrets Provider to abort.
```

Comment:
If these doc updates are for M1 GA, should they include the ability to specify only a directory, where SP will use the group name as a default filename? ([unit test for this case](https://github.com/cyberark/secrets-provider-for-k8s/blob/main/pkg/secrets/pushtofile/secret_group_test.go#L289)) I'm not sure if this is out of scope for the story you're working on.

Reply:
Yeah, this is missing details. I've updated the description to reflect current behavior (i.e. a directory can be specified for file path if custom template is not being used).